### ISSUE TITLE
Scenarios that use _conversation are not allowed to run in parallel

### ISSUE THREAD

**Issue by jimkyndemeyer** [promptfoo/promptfoo#385]
Using `_conversation` in scenario prompts forces concurrency to 1, but I'd like to run separate scenario chats in parallel.

Possibly relates to #384

**Comment by idobenamram** [comment]
i don't think this is solved, when i run with _converstation it still makes everything have a concurrency of 1. is no one using this feature, how is this possible?
